### ISSUE TITLE
Not getting last included file in parser.includes

### ISSUE THREAD

**Issue by sbruno** [plyara/plyara#28]
After parsing a file with only includes, I check the `parser.inlcudes` attribute and the last item is not there:

```
>>> import plyara
>>> parser = plyara.Plyara()
>>> parser.parse_string('include "file1.yara"\ninclude "file2.yara"\ninclude "file3.yara"')
[]
>>> parser.includes
['file1.yara', 'file2.yara']
```

**Comment by rshipp** [comment]
Thanks for the issue!

I'm guessing this is a problem with the grammar, but I don't understand it well enough to find the fix. This applies to both includes and imports, and only happens when there is no rule in the input. The parser is hitting `p_error` with a `p` of `None`.

Here's some debug output:

```
Generating LALR tables
WARNING: 6 shift/reduce conflicts
WARNING: 2 reduce/reduce conflicts
WARNING: reduce/reduce conflict in state 5 resolved using rule (imports_and_scopes -> imports)
WARNING: rejected rule (includes -> imports) in state 5
WARNING: reduce/reduce conflict in state 8 resolved using rule (imports_and_scopes -> includes)
WARNING: rejected rule (imports -> includes) in state 8
Matched import: file1.yara
Matched import: file2.yara
```

<details><summary>Referenced states 5 and 8</summary>

```
state 5

    (4) imports_and_scopes -> imports .
    (7) imports_and_scopes -> imports . scopes
    (10) imports -> imports . import
    (14) includes -> imports .
    (18) scopes -> . scopes scope
    (19) scopes -> . scope
    (16) import -> . IMPORT STRING
    (25) scope -> . PRIVATE
    (26) scope -> . GLOBAL

  ! reduce/reduce conflict for RULE resolved using rule 4 (imports_and_scopes -> imports .)
  ! shift/reduce conflict for IMPORT resolved as shift
  ! shift/reduce conflict for PRIVATE resolved as shift
  ! shift/reduce conflict for GLOBAL resolved as shift
    RULE            reduce using rule 4 (imports_and_scopes -> imports .)
    INCLUDE         reduce using rule 14 (includes -> imports .)
    IMPORT          shift and go to state 12
    PRIVATE         shift and go to state 3
    GLOBAL          shift and go to state 7

  ! IMPORT          [ reduce using rule 14 (includes -> imports .) ]
  ! PRIVATE         [ reduce using rule 14 (includes -> imports .) ]
  ! GLOBAL          [ reduce using rule 14 (includes -> imports .) ]
  ! RULE            [ reduce using rule 14 (includes -> imports .) ]

    import                         shift and go to state 18
    scope                          shift and go to state 9
    scopes                         shift and go to state 17


state 8

    (5) imports_and_scopes -> includes .
    (8) imports_and_scopes -> includes . scopes
    (11) imports -> includes .
    (13) includes -> includes . include
    (18) scopes -> . scopes scope
    (19) scopes -> . scope
    (17) include -> . INCLUDE STRING
    (25) scope -> . PRIVATE
    (26) scope -> . GLOBAL

  ! reduce/reduce conflict for RULE resolved using rule 5 (imports_and_scopes -> includes .)
  ! shift/reduce conflict for INCLUDE resolved as shift
  ! shift/reduce conflict for PRIVATE resolved as shift
  ! shift/reduce conflict for GLOBAL resolved as shift
    RULE            reduce using rule 5 (imports_and_scopes -> includes .)
    IMPORT          reduce using rule 11 (imports -> includes .)
    INCLUDE         shift and go to state 2
    PRIVATE         shift and go to state 3
    GLOBAL          shift and go to state 7

  ! PRIVATE         [ reduce using rule 11 (imports -> includes .) ]
  ! GLOBAL          [ reduce using rule 11 (imports -> includes .) ]
  ! RULE            [ reduce using rule 11 (imports -> includes .) ]
  ! INCLUDE         [ reduce using rule 11 (imports -> includes .) ]

    scopes                         shift and go to state 20
    include                        shift and go to state 21
    scope                          shift and go to state 9
```
</details>


@plyara/all any ideas?

**Comment by Taskr** [comment]
@rshipp Looks like an issue due to some ambiguous grammar specifications. Will take a look when I get chance.

**Comment by rshipp** [comment]
Published v1.3.1 to PyPI, with @Taskr's fix for this issue.

@sbruno feel free to reopen or open a new issue if you find anything else!
